README.md

```diff
@@ -169,6 +169,33 @@ the form `<region>:<table-name>`.
      an MFA token (can only be used with `--role-arn` **and** `--mfa-token`).
    * `--mfa-token` the one-time password generated by your MFA device if assuming a role
      requires an MFA token (can only be used with `--role-arn` **and** `--mfa-serial`).
+   * `--slave-profile`, `--slave-role-arn`, `--slave-mfa-serial`, and `--slave-mfa-token`
+     - see the subheading "Syncing Tables in Two Accounts" for more information.
+
+
+### Syncing Tables in Two Accounts
+
+It is possible to synchronize tables in two accounts. To do so, you must provide
+credentials arguments that configure the credentials providers to be used for the slave
+tables. This is done by providing one of the following combinations of arguments:
+
+   * `--slave-profile` can be used on its own to simply use a profile from your AWS
+     credentials (INI) file for the slave tables.
+   * `--slave-role-arn` can be used on its own to assume a role. When used on its own, it
+     will default to SDK-default credentials to use as its "master credentials" that it
+     uses for permission to assume the role.
+      * It can also be combined with `--slave-profile`, in which case the slave profile
+        will provide the master credentials that are used to assume the role.
+      * It can also be used in conjunction with the `--slave-mfa-serial` and
+        `--slave-mfa-token` arguments if assuming the role requires an MFA token.
+
+By configuring credentials for the slaves using these arguments, the script will use the
+master credentials (provided by default, or by the `--profile`, etc args) to describe and
+read from the master table, and the slave credentials to describe, read from, and write to
+the slave table(s).
+
+See the heading below entitled "Authentication and Authorization to AWS DynamoDB API" for
```

Comment:
I'm personally not a huge fan of linking to headings because I see that break more often than anything ... someone rewords a heading and the links to it break without being noticed. We're talking to developers here ... if they can't find the heading on the page they'll have bigger issues. 😆